scripts/ma_dictionaries.py

```diff
@@ -90,7 +90,7 @@
     "flk": "Falkland Islands (Malvinas)",  # Country
     "fra": "France",  # Country
     "fro": "Faroe Islands",  # Country
-    "fsm": "Micronesia, Federated States of",  # Country
+    "fsm": "Micronesia (Federated States of)",  # Country
```

Comment:
I don't think that this fix is OK supported in future if other datasets are added. But as for now we can leave it as it is